deployment/helm_chart/opik/values.yaml

```diff
@@ -96,7 +96,8 @@ component:
         items:
         - key: default.conf
           path: default.conf
-
+    annotations:
```

Comment:
@obezpalko this should be added only on our environments, not in default values

Reply:
agree